CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 Starting with this release, installing VAST on any Linux becomes
+  significantly easier: A static binary will be provided with each release on
+  the GitHub releases page.
+  [#966](https://github.com/tenzir/vast/966)
```

Comment:
```suggestion
- 🎁 Starting with this release, installing VAST on any Linux becomes
  significantly easier: A static binary will be provided with each release on
  the GitHub releases page. [#966](https://github.com/tenzir/vast/pull/966)
```

Notice the missing `pull` in the URL. Also, this fits the same line with the usual width.